### ISSUE TITLE
Pidgin crashes on send message

### ISSUE THREAD

**Issue by denovoid** [EionRobb/icyque#39]
As of yesterday my Pidgin (2.13.0) crashes each time I send a message through ICQ.
This occurs whether I send a message from the computer or from the ICQ app on my phone.
Is anyone else experiencing this problem?  I suspect some change at ICQ's side.

**Comment by Legodev** [comment]
Have the same issue and updating to master caused me to get "Failed to acquire authentication token (robusto)."

**Comment by denovoid** [comment]
I am using the Windows pre-compiled version -- is there anything I can do ?

**Comment by EionRobb** [comment]
@denovoid can you get a backtrace? https://developer.pidgin.im/wiki/GetABacktrace 

@Legodev are you saying that master does or doesn't crash? That commit was merged into master already

**Comment by Legodev** [comment]
> @Legodev are you saying that master does or doesn't crash? That commit was merged into master already

Master does NOT crash.

**Comment by EionRobb** [comment]
@Legodev "and the robusto issue is not present"? So master works ok for that too?

**Comment by Legodev** [comment]
with the robusto issue is present in master and was introduced with the merge:
8347afc70eb71e3a4bb139a62faa7024ba9070db

side note: i have an old icq account without any phone number/sms/what ever attached. it works with the state before the merge 8347afc70eb71e3a4bb139a62faa7024ba9070db

**Comment by Legodev** [comment]
@EionRobb i found the robusto issue on my end. 
State of master (for me): No crash and no robusto issue.
Sorry for being leading you on a possible wrong track! :(

My issue was:
After reading the commit that caused the robusto issue for me, i noticed that there are many new interactions with the time for example the usage of `ia->server_time_offset`.
Therefore i figured it might be a time related issue on my end...

In short: 
The robusto issue was caused by broken timezone settings, which lead to that some applications, the clock of my desktop environment was not part of it, were not able to read the correct time.

**Comment by denovoid** [comment]
@EionRobb  
Today, in finding the pidgin.RPT file I noticed a number of references in the Call stack list there to 
...\plugins\Copy of libicyque.dll 
such as 
65484086 C:\Program Files\Pidgin\plugins\Copy of libicyque.dll  icq_process_event  C:\cygwin\home\Eion\icyque/libicyque.c:1011

When I previously updated the libicyque.dll file in June 2019 to the current 257,774 byte version, I renamed the previous 251,681 byte version to 'Copy of libicyque.dll' and left it in the plugins folder. I imagined dropping in the new file of the same name would be sufficient to have it used, but from the RPT file it seems the program has still been referencing the old file somehow, even though renamed, for the best part of a year, with no previous ill-effects.

So I have just now removed the 'Copy of...' file, and re-run Pidgin and after a hiccuped first attempt, it seemed to pick up the new libicyque.dll and the ICQ crashing issue has now disappeared.

All sent messages are now still subject to showing twice in the client window, whereas before it was only sometimes, but that's another issue.

Thanks for pointing me at trying to get a backtrace :)
